packages/vite-config/package.json

```diff
@@ -34,15 +34,13 @@
     }
   },
   "devDependencies": {
-    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
-    "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
```

Comment:
These are not maintained anymore. Note we still use them to create the UMD bundle for synapse-react-client